doc/modules/ensemble.rst

```diff
@@ -876,7 +876,7 @@ controls the number of iterations of the boosting process:
   >>> clf = HistGradientBoostingClassifier(max_iter=100).fit(X_train, y_train)
 
   >>> clf.score(X_test, y_test)
-  0.8998
+  0.8965
```

Comment:
Why does this PR change the accuracy when there is no missing data?

Reply:
Because we have one less bin